### DISCUSSION TITLE
Implementing IP Pool Feature

### DISCUSSION THREAD

**Discussion by yahavf6** [postalserver/postal#2283] [Help with using Postal]
Hey,

How do I use the Postal IP Pool feature?
I have enabled and assigned it to server correctly, Specifically, I need help with buying the correct proxies, configuring my DNS to use them, and authenticating with the proxy provider to make it work all work.
Can you provide some guidance on these steps?

**Comment by willpower232** [comment]
Do you already have multiple IP addresses from your server provider? I'm not sure what you mean "buying the correct proxies"

**Reply by yahavf6** [reply]
I'm using digital ocean, I don't think they allow multiple IP's for the same droplet... I thought to use reverse proxy using NGINX to do this but according to your answer, there's no way to do it.
Do you have any recommendations on where to host my Postal server to achieve multiple IPs?

**Reply by willpower232** [reply]
Even if you had some proxies, you can't magically get additional IP addresses without your server providers assistance. Each server provider has their own arrangements for these things so I can't really recommend one over the other.
